loader/src/ui/mods/list/ModItem.cpp

```diff
@@ -379,7 +377,14 @@ void ModItem::updateState() {
             on->setColor(ccGRAY);
             on->setOpacity(105);
         }
+
+        // Make sure toggler is always the same size and position to prevent misalignment
+        m_enableToggle->setContentSize({35, 35});

```

Comment:
`CCMenuItemToggler::createWithStandardSprites` should just set the content size of the selected sprite to the same as the unselected sprite for a systematic fix of the root issue

Reply:
That is what this uses, but it doesn't work, causing the checkbox to shift in the layout when the content size changes